pages/mesosphere/dcos/services/hdfs/2.6.0-3.2.0/release-notes/index.md

```diff
@@ -3,11 +3,43 @@ layout: layout.pug
 navigationTitle: Release Notes
 excerpt: Discover the new features, updates, and known limitations in this release of the HDFS Service
 title: Release Notes
-menuWeight: 10
+menuWeight: 0
 model: /mesosphere/dcos/services/hdfs/data.yml
 render: mustache
 ---
 
+# Release Notes for HDFS Service version 2.6.0-3.2.0
+
+## Feature
+- Added support for custom domains. See [custom domains](/mesosphere/dcos/services/{{ model.serviceName }}/2.6.0-3.2.0/operations/security/#forwarding-dns-and-custom-domain)
+- Added support for DC/OS Storage Service (DSS). See [DSS documentation](https://docs.d2iq.com/mesosphere/dcos/services/storage/1.0.0/)
+- Added marathon service scheduler checks
+
+## Bug Fixes
+- Fixed a bug in Apache Derby related to the miscalculation of bytes read for a principal entry in the Kerberos keytab file. See [HADOOP-16283](https://issues.apache.org/jira/browse/HADOOP-16283)).
+
+## Updates
```

Comment:
Can you move Updates above Features? It follows the same pattern as our other packages e.g
- cassandra: https://docs.d2iq.com/mesosphere/dcos/services/cassandra/2.6.0-3.11.4/release-notes/
- kafka: https://docs.d2iq.com/mesosphere/dcos/services/kafka/2.6.0-2.2.1/release-notes/

Reply:
Done